extension.schema.json

```diff
@@ -854,6 +854,12 @@
           "items": { "$ref": "#/definitions/contentActionRef" },
           "minItems": 1
         },
+        "bulk-actions-dropdown": {
```

Comment:
This should be a generic feature since we will add new bulk actions in the future so I would call it `bulk-actions`

Reply:
these actions appears in dropdown only. But it's ok for me to rename

Reply:
For now but this is supposed to be the generic feature for loading bulk actions, in future they might be used in other places too, please keep that in mind

Reply:
renamed to `bulk-actions` and remove `component` property